migrations/0014_create_events_v2.up.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS events;
+
+CREATE TABLE events (
+    id INTEGER PRIMARY KEY NOT NULL,
+    title TEXT NOT NULL,
+    slug TEXT NOT NULL,
+    description TEXT NOT NULL,
+    flyer TEXT,
```

Comment:
is this a url? `flyer_url` maybe?

Reply:
probably gonna remove this and add file uploads in a separate PR